init-tools.sh

```diff
@@ -157,7 +157,7 @@ if [ ! -e $__INIT_TOOLS_DONE_MARKER ]; then
     Tools-Override/crossgen.sh $__scriptpath/Tools
 
     echo "Making all .sh files executable under Tools."
-    ls $__scriptpath/Tools/*.sh | xargs chmod +x
+    find $__scriptpath/Tools/ -name "*.sh" -exec chmod +x {} \;
```

Comment:
Hm, unfortunately, that only gets finds the "top-level" sh files for me, and nothing in nested directories...